README.md

```diff
@@ -3,27 +3,29 @@
 Zed offers a new approach to data that makes it easier to manipulate and manage
 your data.
 
-With Zed's new [super-structured data model](docs/formats/README.md#2-zed-a-super-structured-pattern),
+With Zed's new
+[super-structured data model](https://zed.brimdata.io/docs/formats/#2-zed-a-super-structured-pattern),
 messy JSON data can easily be given the fully-typed precision of relational tables
 without giving up JSON's uncanny ability to represent eclectic data.
 
-Trying out Zed is easy: just [install](#quick-start) the command-line tool
-[`zq`](docs/commands/zq.md).
+Trying out Zed is easy: just [install](https://zed.brimdata.io/docs/#getting-started)
```

Comment:
should this point to the installation instructions instead?
```suggestion
Trying out Zed is easy: just [install](https://zed.brimdata.io/docs/install/)
```

Reply:
This PR just moves the links.  Let's muck with install in a subsequent PR.  Also, I want to move the python docs to docs/libraries/python.md and also create a go.md with initial boilerplate so we have a place to hang dev efforts around cross-language lib support.